client/packages/requisitions/src/RequestRequisition/api/operations.graphql

```diff
@@ -26,6 +26,9 @@ fragment RequestRow on RequisitionNode {
     id
   }
   orderType
+  lines {
+    totalCount
+  }
```

Comment:
This is what's causing the compilation error -- this type is now no longer compatible with `RequestFragment`.